### DISCUSSION TITLE
How does a organization member access a private repo with CLI on a public computer

### DISCUSSION THREAD

**Discussion by Azleal** [community/community#129552] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I'm collaborating in an organization private repo, and I need to `git clone` this repo on a public machine, which  means that, I cannot configure its ssh key to my github account, because anyone else could access this machine and thus get access to my github account. I've tried to use `Personal Access Token`, however, it can only give access to those **Personal** repos. 

I'd appreciate it if anyone please tell me what should I do or do I miss someting. Thanks

**Comment by LEDparty** [comment, marked as answer]
According to this, you can still generate personal access tokens if you are part of an organization:

https://docs.github.com/en/organizations/managing-programmatic-access-to-your-organization/setting-a-personal-access-token-policy-for-your-organization

**Reply by Azleal** [reply]
Thank you for your reply @LEDparty, this post works for me. I didn't know that the organization's default personal access settings are off. I've tried it as instructed it worked.
